index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="container">
+        <a class="logo">My Logo</a>
+
+        <nav class="menu-icon">
+            <img src="src/Icon-Menu.png" alt="menu"/>
+        </nav>
+    </header>
+    <section class="headline">
+
+        <h2 class="title">I’m Mariane Glad to see you!
+            </h2>
+
+        <p class="paragraph">I’m a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+
+        <p class="line">LET'S CONNECT</p>
+
+        
+        <ul class="social-media-icons">
+                <li><a href=""><img class="icon twitter" 
+                    src="src/Twitter.png" alt="twitter-icon"/></a></li>
+
+                <li><a href=""><img class="icon linkedin" 
+                    src="src/Linkedin-icon.png" alt="linkedin-icon"/></a></li>
+
+                <li><a href=""><img class="icon msign" 
+                    src="src/m.png" alt="msign"/></a></li>
+
+                <li><a href=""><img class="icon github" 
+                    src="src/Github.png" alt="github-icon"/></a></li>
+
+                <li><a href=""><img class="icon finger" 
+                    src="src/Angel.png" alt="Angel-List"/></a></li>
+        </ul>
```

Comment:
- [ ] Peeeeerfect 😄 ! you did a great job on wrapping those in a ul tag, however, this alignment is yet well arranged, see this image below and fix all as required, please.
- [ ] One more thing is that, considering the Figma design, kindly adjust your margins, paragraph color, font-size and padding to fit the exact template, I will drop and image soon 😄 
- [ ] You background image is not covering the section as it should, please adjust your properties to fit the original design.
<img width="303" alt="h12" src="https://user-images.githubusercontent.com/79058364/154265324-b9924a9e-2976-41fd-9585-9383e3b1e9cc.png">

### consider making those changes to make design look like so ⬇️ 

<img width="258" alt="hd" src="https://user-images.githubusercontent.com/79058364/154264470-87c4d327-3731-470c-a57f-6c982c151eb6.png">